.github/workflows/tests.yml

```diff
@@ -32,7 +32,12 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install -e ".[dev,all_models]"
+          python -m pip install -e ".[dev]"
+          python -m pip install xgboost
+          python -m pip install spacy
+          python -m pip install torch
+          python -m pip install statsmodels
```

Comment:
this is effectively the same as `python -m pip install -e ".[dev,all_models]"`, but for some reason Python 3.7 will not complete installation, ending in an `OSError` saying there is not enough space 😩